tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/RealPath.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.Sdk.Tools.Cli.Helpers;
+
+public static class RealPath
+{
+    public static string GetRealPath(string path, int maxDepth = 64)
+    {
+        if (string.IsNullOrWhiteSpace(path))
+        {
+            throw new ArgumentException("Path is null or empty.", nameof(path));
+        }
+
+        var full = Path.GetFullPath(path);
+        var sep = Path.DirectorySeparatorChar;
+
+        string root = Path.GetPathRoot(full)!;
+        string remainder = full.Substring(root.Length);
+        string[] parts = remainder.Split(new[] { '\\', '/' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
```suggestion
        string[] parts = remainder.Split(Path.DirectorySeparatorChar, StringSplitOptions.RemoveEmptyEntries);
```

Reply:
outdated, please review the latest version